addons/goggles/functions/fnc_onEachFrame.sqf

```diff
@@ -14,6 +14,7 @@
  * Public: No
  */
 #include "script_component.hpp"
+if !(hasInterface) exitWith {};
```

Comment:
Don't add the PFH on a dedi/hc. Seems better than to check something every frame that will never change.